src/frontend/src/flows/authorize/postMessageInterface.ts

```diff
@@ -1,5 +1,6 @@
 // Types and functions related to the window post message interface used by
 // applications that want to authenticate the user using Internet Identity
+import { zodPrincipal } from "@dfinity/internet-identity-vc-api";
```

Comment:
Should we put it somewhere else? `zodPrincipal` doesn't seem related to `internet-identity-vc-api` to me. I don't think I would remember to import it from here.

Maybe not in this PR though.

Reply:
Yeah, I thought about it. But then we need to have yet another package that only provides the `zodPrincipal`. Maybe we could put that into `ic-js` AFAIK @peterpeterparker also needs this definition in the signer library.

Reply:
Seems weird there as well. I was thinking more in II directly. I think that repeating this code in multiple places is also ok.